c_emulator/riscv_sim.c

```diff
@@ -1022,7 +1022,8 @@ int main(int argc, char **argv)
   // Initialize model so that we can check or report its architecture.
   preinit_sail();
 
-  char *file = process_args(argc, argv);
+  int files_start = process_args(argc, argv);
+  char *file0 = argv[files_start];
```

Comment:
Please use a better name than 'file0' for this variable (and possibly also add a comment).  It is not clear from the name (i.e., without reading the rest of the change) that this is assumed to be the filename of the executable (and that any further filenames should be 'additional' ELF files).

Reply:
Sure, how about 'initial_file' or 'main_file'?